bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/thing/server-thing-type.xml

```diff
@@ -23,10 +23,22 @@
 			<channel-group typeId="connection-status" id="connection-status"/>
 			<channel-group typeId="ftth" id="ftth"/>
 			<channel-group typeId="xdsl" id="xdsl"/>
+			<channel-group typeId="wireguard" id="vpn-server">
+				<label>Wireguard VPN</label>
+			</channel-group>
+			<channel-group typeId="openvpn-bridged" id="vpn-server">
+				<label>OpenVPN Bridged VPN</label>
+			</channel-group>
+			<channel-group typeId="openvpn-routed" id="vpn-server">
+				<label>OpenVPN Routed VPN</label>
+			</channel-group>
+			<channel-group typeId="pptp" id="vpn-server">
+				<label>PPTP VPN</label>
+			</channel-group>
 		</channel-groups>
 
 		<properties>
-			<property name="thingTypeVersion">1</property>
+			<property name="thingTypeVersion">2</property>
```

Comment:
thing type version is increased, but the actual instructions are missing

Reply:
I know, it's in the Todo. Upgrade instructions will be added once channels are stabilized.